client/app/project/[slug]/page.tsx

```diff
@@ -204,12 +156,12 @@ export default function ProjectPage() {
       <div className="container mb-16" id="contributors">
         <h2 className="text-3xl font-bold mb-8">Աջակիցներ</h2>
         <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-2 gap-6">
-          {projectData.contributors.map((contributor, index) => (
+          {project.blogs[0].contribution.map((contributor, index) => (
```

Comment:
We need to fix this, here should be a list of ENV amount of contributors from across the blogs

Reply:
Let's do it in next PRs, I need to connect all project to make it easier. I have left TODO comment here